GitUI/CommandsDialogs/FormClone.cs

```diff
@@ -363,8 +365,6 @@ private void ToSelectedIndexChanged(object sender, EventArgs e)
             ToTextUpdate(sender, e);
         }
 
-        private readonly AsyncLoader _branchListLoader = new();
```

Comment:
❓ Why did you start to replace AsyncLoader in this anyway huge PR?

Reply:
It was a reluctant change. `Module.GetRemoteServerRefs` was only used in FormClone, and there was no reason to expose this API via the `IGitModule`.